DeploymentAgentReconfigurationScripts/TFS2018 Update2/HardwareMoveRemoveOldAgent.ps1

```diff
@@ -0,0 +1,178 @@
+<# 
+This is a guidance script to bring a deployment group agent online after TFS Hardware move activity for TFS2018 Update2. Post Hardware move, all the agents will be offline as the TFS server url has been updated. User can use\follow this script to bring them online. The script has to be run from Administrator PowerShell prompt on the each agent machine.
```

Comment:
Should this script be specific to deployment group agent? Can we take the agent type as input and handle for both automation and deployment group agents?

Reply:
No. I will prefer a different script script for automation agent and we probably not maintain that script in our repo.